javascripts/controllers/EditCtrl.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+app.controller("EditCtrl", function($location, $rootScope, $routeParams, $scope, ContactService) {
+	
+const getContacts = () => {
+		ContactService.getOneContact($routeParams.id).then((results) => {
+			$scope.contact = results.data;
```

Comment:
This line isn't actually working. It should be loading the contact information you are pulling back here into the fields in the edit partial.

Reply:
Edit fields have contact info now and update the contact